Sources/BeeTracking/ProbabilisticTracker.swift

```diff
@@ -5,6 +5,69 @@ import TensorFlow
 import PythonKit
 import Foundation
 
+/// Runs the random projections tracker
+/// Given a training set, it will train an RP tracker
+/// and run it on one track in the test set:
+///  - output: image with track and overlap metrics
+public func runProbabilisticTracker<
+  Encoder: AppearanceModelEncoder,
```

Comment:
Why not just template on likelihood model?

Reply:
The problem is likelihood model is a struct, so it cannot be a template argument. I think we need to talk on this.